api/v1alpha4/openstackmachine_types.go

```diff
@@ -90,6 +90,8 @@ type OpenStackMachineSpec struct {
 	// IdentityRef is a reference to a identity to be used when reconciling this cluster
 	// +optional
 	IdentityRef *OpenStackIdentityReference `json:"identityRef,omitempty"`
+
+	AdminPass string `json:"adminPass,omitempty"`
```

Comment:
Having this password in the Resources might conflict on some security requirements. What do you think about reading the password from a secret, similar to `OpenStackCluster.Spec.IdentityRef`?

Reply:
Thanks,this might be a good suggestion